src/content/docs/query-your-data/nrql-new-relic-query-language/get-started/nrql-syntax-clauses-functions.mdx

```diff
@@ -2324,8 +2324,9 @@ Note: `aparse()` is case-insensitive.
     Use `capture()` to extract values from an attribute using a regular expression with [RE2 syntax](https://github.com/google/re2/wiki/Syntax).
 
     It takes two arguments:
-    * Attribute name.
-    * Regular expression with capture syntax. Regex expressions in NRQL use Python-like syntax, `r'...'`.
+    
+      * Attribute name.
```

Comment:
Periods in this list I think aren't needed but might require the "regular expression" line to be reworded slightly